includes/CLI/Plugin_Check_Command.php

```diff
@@ -114,20 +116,18 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
 	 */
 	public function check( $args, $assoc_args ) {
-		/*
-		 * Bail early if the Plugin Checker is not activated.
-		 *
-		 * If the Plugin Checker is not activated, it will throw an error and
-		 * CLI commands won't be usable.
-		 */
-		if ( is_plugin_inactive( $this->plugin_context->basename() ) ) {
```

Comment:
@felixarntz I took liberty to remove this dead code because we were getting `Cyclomatic complexity` issue. Removing this helped reducing that complexity.

Reply:
Not sure why this was here in the first place. Probably safe to remove indeed 👍